src/HomePage.js

```diff
@@ -19,6 +31,7 @@ export function HomePage({ isExpanded, setIsExpanded }) {
           key={datum.post_id}
         />
       ))}
+      <Suggestions isExpanded={true} />
```

Comment:
I find it strange that `isExpanded` is true when you only show 5 items, shouldn't it be the opposite?

Reply:
True 🍡